rules/azure/credential_access_storage_account_key_regenerated.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/08/19"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a rotation to storage account access keys in Azure. Regenerating access keys can affect any applications
+or Azure services that are dependent on the storage account key.
```

Comment:
I'd recommend including a sentence to explain why an adversary might perform this action.

Reply:
👍